src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/GameReports/PivotGameResultViewModel.cs

```diff
@@ -114,39 +114,54 @@ public PivotGameResultViewModel ReverseTeams()
             return result;
         }
 
-        /// <summary>
-        /// According game score returns cascade style sheets class name
-        /// </summary>
-        /// <param name="homeScore">Sets score of home team</param>
-        /// <param name="awayScore">Sets score of away team</param>
-        /// <returns>Name of the class with cascade style sheets settings</returns>
-        private static string GetCssClass(byte? homeScore, byte? awayScore)
+        private static void GetCssClassWhenWin(byte? awayScore, ref string cssClass)
         {
-            string cssClass = CssClassConstants.NORESULT;
-            if (homeScore == THREE && awayScore == ZERO)
+            if (awayScore == ZERO)
             {
                 cssClass = CssClassConstants.WIN_3_0;
             }
-            else if (homeScore == THREE && awayScore == ONE)
+            else if (awayScore == ONE)
             {
                 cssClass = CssClassConstants.WIN_3_1;
             }
-            else if (homeScore == THREE && awayScore == TWO)
+            else if (awayScore == TWO)
             {
                 cssClass = CssClassConstants.WIN_3_2;
             }
-            else if (homeScore == TWO && awayScore == THREE)
+        }
+
+        private static void GetCssClassWhenLoss(byte? homeScore, ref string cssClass)
+        {
+            if (homeScore == TWO)
             {
                 cssClass = CssClassConstants.LOSS_2_3;
             }
-            else if (homeScore == ONE && awayScore == THREE)
+            else if (homeScore == ONE)
             {
                 cssClass = CssClassConstants.LOSS_1_3;
             }
-            else if (homeScore == ZERO && awayScore == THREE)
+            else if (homeScore == ZERO)
             {
                 cssClass = CssClassConstants.LOSS_0_3;
             }
+        }
+        /// <summary>
+        /// According game score returns cascade style sheets class name
+        /// </summary>
+        /// <param name="homeScore">Sets score of home team</param>
+        /// <param name="awayScore">Sets score of away team</param>
+        /// <returns>Name of the class with cascade style sheets settings</returns>
+        private static string GetCssClass(byte? homeScore, byte? awayScore)
```

Comment:
While technically you avoided complex ifs - practically this is hard to understand. Look at game-result-cell.directive.ts file in WebClient project

Reply:
Ok. Thanks for hint